src/components/book-card/z-book-card/index.tsx

```diff
@@ -57,15 +56,23 @@ export class ZBookCard {
   @Prop()
   year?: string;
 
-  /** [optional] Show link to the ebook resource. Set title and cover as clickable, opening a new tab to the ebook resource */
+  /** [optional] Link to the ebook resource. Creates a link also around the cover and title. */
   @Prop()
   ebookUrl?: string;
 
-  /** [optional] Fallback cover URL */
+  /** [optional] Value for the `target` attribute of the links created by `ebookUrl` prop. */
+  @Prop()
+  linkTarget = "_blank";
+
+  /** [optional] Fallback cover URL. */
   @Prop()
   fallbackCover?: string;
 
-  /** [optional] Set a specific h level as html tag for opera title */
+  /** Whether to show a stack of covers to represent multiple books (only for `portrait` variant). */
+  @Prop()
+  hasMultipleCovers = false;
```

Comment:
Questa prop non sembra produrre il risultato atteso, in quanto la copertina "di mezzo" è coperta da quella che dovrebbe stare sotto.